### PR TITLE
Ts 221 modify secure message collection case hardcoded

### PR DESCRIPTION
**Pull request by dcdarrell9**
[Trello Card](https://trello.com/c/zKGpcaUs)

removed hardcoding for collection case in create message

### CODE REVIEW

File: frontstage/views/secure_messaging.py [hunk 1]
```diff
@@ -42,12 +42,24 @@
 def create_message(session):
     """Handles sending of new message"""
 
+    url = app.config['RM_CASE_GET_BY_PARTY'].format(app.config['RM_CASE_SERVICE'], session['user_uuid'])
```
Comment (andrewmil) [inline frontstage/views/secure_messaging.py]: still using `user_uuid` should be `party_id` as you've got @AndrewTorrance changes

File: frontstage/views/secure_messaging.py [hunk 2]
```diff
@@ -42,12 +42,24 @@
...
+    # url = "http://localhost:8050/api/party-api/respondents/id/" + session['user_uuid']
+    collection_response = requests.get(url)
+    if collection_response.status_code != 200:
+        return redirect(url_for('error_bp.error_page'))
+    collection_response_json = collection_response.json()
+    collection_id = collection_response_json[0].get('id')
+    if collection_id:
+        collection_case = collection_id
+    else:
+        collection_case = None
```
Comment (andrewmil) [inline frontstage/views/secure_messaging.py]: Could also have this as a separate function as we're using with reply as well but that could be done in that card.

File: frontstage/views/secure_messaging.py (hunk 1 above)
Comment (dcdarrell9) [inline frontstage/views/secure_messaging.py]: 👍  Apparently the merge didn't work on my terminal.

File: frontstage/views/secure_messaging.py (hunk 2 above)
Comment (dcdarrell9) [inline frontstage/views/secure_messaging.py]: Chuck it into the other card, just in case there is anything different.

**Pull request merged**